controllers/user.controller.js

```diff
@@ -0,0 +1,70 @@
+const User=require("../models/user-model");
+
+const getUsers=async(req,res)=>{
+  try{
+    let users=await User.find({})
+    users=users.map((user)=>{
+      user.password=undefined;
+      return user;
+    })
+    res.json({success:true , users})
+  }catch(err){
+    console.log(err.message)
+    res.status(500).json({
+      success:false,
+      message:"Unable to get Users list",
```

Comment:
Why do we need to fetch all users Kunal ?

Reply:
Yes, thanks just trying to test something. Will remove it